### PR TITLE
WIP: Add new Section for Grid 4

### PR DESCRIPTION
**Pull request by raju249**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

**Comment by Harsha509** [comment]
Hi @raju249,

Thanks for the PR!

1. Is the PR is still WIP ? or ready for review ? ( As title describes WIP and the PR is not a draft so asking this)
2. Can you please complete the checklist and types of changes above

Thanks,
Harsha.

File: docs_source_files/content/grid 4/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+and allows control of these instances.
+
+Selenium Grid 4 allows us to run tests in parallel on multiple machines,
+and to manage different browser versions and browser configurations centrally
+(instead of in each individual test).
+
+Selenium Grid 4 is not a silver bullet.
+It solves a subset of common delegation and distribution problems,
+but will for example not manage your infrastructure,
+and might not suit your specific needs.
```
Comment (Harsha509) [inline docs_source_files/content/grid 4/_index.en.md:27]: Hi @raju249 ,

Can you please changes in other tanslated pages other than en (like es,fr,de etc)

Thanks,
 Harsha

**Comment by raju249** [comment]
Hey @Harsha509 yes, this is WIP PR. I just made the changes yesterday, and yes I will be adding files for other language translations too.

Sorry, I did not create this as a draft PR. 🙏

**Comment by Harsha509** [comment]
> Hey @Harsha509 yes, this is WIP PR. I just made the changes yesterday, and yes I will be adding files for other language translations too.
> 
> Sorry, I did not create this as a draft PR. 🙏

Hi @raju249,

Sure. Thanks for the info.

Regards,
Harsha

**Comment by raju249** [comment]
Hey @diemol 

I am yet to add the doc for using jdbc backed session map, but that is on hold since I am yet to figure out getting it worked with a jdbc driver, it has some issues.

But do you think I can just a `Grid 4` section and we can merge this PR?

**Comment by manoj9788** [comment]
I have a pictorial representation of Selenium 4 Grid deployment here: https://applitools.com/blog/selenium-4/ Feel free to use it!

**Comment by raju249** [comment]
Thanks @manoj9788 

I am working on this PR today, will try to wrap it up.

**Comment by raju249** [comment]
I wasn't aware of the base branch `dev` 😞 

I am closing this PR, raising a new one with changes.

**Pull request closed without merging**